modules/dns_nsupdate/dns_nsupdate_gss_plugin.rb

```diff
@@ -0,0 +1,17 @@
+module ::Proxy::Dns::NsupdateGSS
+  class Plugin < ::Proxy::Provider
+    plugin :dns_nsupdate_gss, ::Proxy::VERSION,
+           :factory => proc { |attrs| ::Proxy::Dns::NsupdateGSS::Record.record(attrs) }
+
+    default_settings :dns_server => 'localhost', :dns_ttl => 86_400,
```

Comment:
Not sure I understand you. Are you asking whether the `dns_ttl` default value should refer to the one defined in `Proxy::Dns::Plugin`? I suppose that's acceptable in this case -- providers exist in the context of the main module, and I think such a dependency is ok.